pdf-core.gemspec

```diff
@@ -2,7 +2,7 @@
 
 Gem::Specification.new do |spec|
   spec.name = 'pdf-core'
-  spec.version = '0.8.0'
+  spec.version = '0.8.1'
```

Comment:
This doesn't belong here. I'll take a closer look at the rest later.

Reply:
Oh sorry, yes I just needed that so that I could use the fork in my Gemfile. Will remove from this branch